pkgs/development/haskell-modules/configuration-hackage2nix/broken.yaml

```diff
@@ -744,12 +744,6 @@ broken-packages:
   - chakra # failure in job https://hydra.nixos.org/build/282936173 at 2024-12-24
   - chalkboard # failure in job https://hydra.nixos.org/build/234453414 at 2023-09-13
   - chalmers-lava2000 # failure in job https://hydra.nixos.org/build/233239592 at 2023-09-02
-  - changelog-d # failure in job https://hydra.nixos.org/build/252716175 at 2024-03-16
-  - changelog-d # failure in job https://hydra.nixos.org/build/253689337 at 2024-03-31
-  - changelog-d # failure in job https://hydra.nixos.org/build/255671571 at 2024-04-16
-  - changelog-d # failure in job https://hydra.nixos.org/build/257082502 at 2024-04-27
-  - changelog-d # failure in job https://hydra.nixos.org/build/259607598 at 2024-05-15
-  - changelog-d # failure in job https://hydra.nixos.org/build/259957724 at 2024-05-19
   - changelog-d # failure in job https://hydra.nixos.org/build/260724402 at 2024-05-25
```

Comment:
You can remove this altogether. changelog-d is a package that is not defined in `hackage-packages.nix`, so these entries get generated erroneously (we should probably attempt to fix this).